assets/xml/objects/gameplay_keep.xml

```diff
@@ -1,369 +1,951 @@
+        <Texture Name="gDropMagicLargeTex" OutName="drop_magic_large" Format="rgb5a1" Width="32" Height="32" Offset="0x00041940"/>
+        <Texture Name="gRupeeGreenTex" OutName="rupee_green" Format="rgb5a1" Width="4" Height="4" Offset="0x00042140"/>
+        <Texture Name="gRupeeBlueTex" OutName="rupee_blue" Format="rgb5a1" Width="4" Height="4" Offset="0x00042160"/>
+        <Texture Name="gRupeeRedTex" OutName="rupee_red" Format="rgb5a1" Width="4" Height="4" Offset="0x00042180"/>
+        <Texture Name="gRupeePinkTex" OutName="rupee_pink" Format="rgb5a1" Width="4" Height="4" Offset="0x000421C0"/>
+        <Texture Name="gRupeeOrangeTex" OutName="rupee_orange" Format="rgb5a1" Width="4" Height="4" Offset="0x000421A0"/>
+        <Texture Name="gRupeeSilverTex" OutName="rupee_silver" Format="rgb5a1" Width="4" Height="4" Offset="0x000421E0"/>
+        <DList Name="gRupeeDL" Offset="0x00042440"/>
+        <Texture Name="gRupeeLightRedTex" OutName="rupee_light_red" Format="rgb5a1" Width="4" Height="4" Offset="0x00042510"/>
+        <Texture Name="gUnknownStone1Tex" OutName="unknown_stone_1" Format="rgb5a1" Width="32" Height="32" Offset="0x00042530"/>
+        <Texture Name="gUnknownStone2Tex" OutName="unknown_stone_2" Format="rgb5a1" Width="32" Height="32" Offset="0x00042D30"/>
+        <Texture Name="gUnusedTreeStumpTopTex" OutName="unused_tree_stump_top" Format="rgb5a1" Width="32" Height="32" Offset="0x00043530"/>
+        <Texture Name="gUnusedTreeStumpBarkTex" OutName="unused_tree_stump_bark" Format="rgb5a1" Width="32" Height="32" Offset="0x00043D30"/>
+        <Texture Name="gUnusedGrassBladesTex" OutName="unused_grass_blades" Format="rgb5a1" Width="64" Height="32" Offset="0x00044530"/>
+        <Texture Name="gUnusedStoneHookshotTargetTex" OutName="unused_stone_hookshot_target" Format="rgb5a1" Width="32" Height="32" Offset="0x00045530"/>
+        <Texture Name="gUnknownStone3Tex" OutName="unknown_stone_3" Format="rgb5a1" Width="32" Height="32" Offset="0x00045D30"/>
+        <Texture Name="gSignRectangularSideTex" OutName="sign_rectangular_side" Format="rgb5a1" Width="32" Height="16" Offset="0x00046530"/>
+        <Texture Name="gSignRectangularFrontTex" OutName="sign_rectangular_front" Format="rgb5a1" Width="32" Height="16" Offset="0x00046930"/>
+        <Texture Name="gSignPostWoodTex" OutName="sign_post_wood" Format="rgb5a1" Width="64" Height="8" Offset="0x00046D30"/>
+        <Texture Name="gSignLetteringTex" OutName="sign_lettering" Format="rgb5a1" Width="64" Height="32" Offset="0x00047130"/>
+        <DList Name="gRaindropDL" Offset="0x00048160"/>
+        <DList Name="gFootprintShadowDL" Offset="0x00048180"/>
+        <Texture Name="gFootprintShadowTex" OutName="footprint_shadow" Format="ia16" Width="32" Height="64" Offset="0x00048210"/>
+        <DList Name="gCircularShadowDL" Offset="0x00049210"/>
+        <Texture Name="gCircularShadowTex" OutName="circular_shadow" Format="ia16" Width="32" Height="32" Offset="0x00049290"/>
```

Comment:
we call it a circle shadow in the code, less of a mouthful than circular imo

Reply:
thats fine